web/modules/custom/dpl_opening_hours/dpl_opening_hours.routing.yml

```diff
@@ -0,0 +1,10 @@
+dpl_opening_hours.content:
+  path: "/node/{node}/edit/opening-hours"
+  defaults:
+    _controller: '\Drupal\dpl_opening_hours\Controller\OpeningHoursController::content'
+    _title: "Opening Hours"
+  requirements:
+    _permission: "access admin tab"
+    _custom_access: '\Drupal\dpl_opening_hours\Controller\OpeningHoursController::adminTabAccess'
```

Comment:
See overall reply, this is related to that. 

```
 requirements:
    _permission: 'access opening hours tab'
    
    ```

Reply:
I have added a new permission for handling this and added it to the two administrator roles.